VotingApplication/VotingApplication.Web/Api/Controllers/PollChoiceController.cs

```diff
@@ -56,6 +57,8 @@ public void Post(Guid pollId, ChoiceCreationRequestModel choiceCreationRequest)
                 poll.LastUpdatedUtc = DateTime.UtcNow;
 
                 context.SaveChanges();
+
+                ClientSignaller.SignalUpdate(poll.UUID.ToString());
```

Comment:
Signalling identifiers are all strings. Strictly speaking, they are the name of the signalR group group.